pkgs/applications/radio/chirp/default.nix

```diff
@@ -10,13 +10,13 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "chirp";
-  version = "0.4.0-unstable-2024-05-24";
+  version = "0.4.0-unstable-2024-07-05";
```

Comment:
```suggestion
  version = "0.4.0-unstable-2024-07-05";
  pyproject = true;
```

Reply:
Adding the following to chirp causes the build to fail to find the `suds` package: 

```
pyproject = true;
build-system = with python3.pkgs; [ setuptools ];
```

Any objection to holding off on non-update related changes to the `chirp` package till after this PR?

Reply:
In the interest of moving the core changes forward I am going to resolve these and plan on a follow-up PR once the chirp package is updated/stable again.